.github/workflows/test-config.yml

```diff
@@ -14,9 +14,9 @@ jobs:
       fail-fast: true
       matrix:
         php: [ '7.4', '8.0', '8.1' ]
-        stability: [ prefer-stable ]
+        composer_flags:  ['', '--prefer-lowest']
```

Comment:
`--prefer-lowest` is failing 😢  https://github.com/DarkaOnLine/L5-Swagger/runs/4892627273?check_suite_focus=true

Reply:
True, `composer install` doesn't support that flag, only `composer update` does. Since we don't have a `composer.lock` file in this repository, we can safely run update instead of install.

Fixed by e429073.